### PR TITLE
Add more tests for registering

### PR DESCRIPTION
**Pull request by mkdesignn**

### CODE REVIEW

File: src/main/java/com/example/jwt/service/UserServiceImp.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.example.jwt.service;
+
+import com.example.jwt.entity.User;
+
+public interface UserServiceImp {
+
+    User RegisterUser(User user);
+}
```
Comment (Imany-apk) [inline src/main/java/com/example/jwt/service/UserServiceImp.java]: Impl Postfix is for the Implementation class not for the interface.

Comment (mkdesignn) [reply src/main/java/com/example/jwt/service/UserServiceImp.java]: I'll fix it today, thanks.

File: src/main/java/com/example/jwt/service/UserServiceImp.java [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
 
 import com.example.jwt.entity.User;
+import com.example.jwt.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
 
-public interface UserServiceImp {
+@Component
+@RequiredArgsConstructor
+public class UserServiceImp implements UserServiceImp {
```
Comment (Imany-apk) [inline src/main/java/com/example/jwt/service/UserServiceImp.java]: you forgot it, it should implement UserService not it's own self.

Comment (mkdesignn) [reply src/main/java/com/example/jwt/service/UserServiceImp.java]: Sorry, I fucked that part

**Pull request merged**